src/data/text/move_descriptions.h

```diff
@@ -3741,4 +3741,28 @@ const u8 *const gMoveDescriptionPointers[MOVES_COUNT - 1] =
     [MOVE_GLACIAL_LANCE - 1] = sGlacialLanceDescription,
     [MOVE_ASTRAL_BARRAGE - 1] = sAstralBarrageDescription,
     [MOVE_EERIE_SPELL - 1] = sEerieSpellDescription,
+    [MOVE_DIRE_CLAW - 1] = gNotDoneYetDescription,
+    [MOVE_PSYSHIELD_BASH - 1] = gNotDoneYetDescription,
+    [MOVE_POWER_SHIFT - 1] = gNotDoneYetDescription,
+    [MOVE_STONE_AXE - 1] = gNotDoneYetDescription,
+    [MOVE_SPRINGTIDE_STORM - 1] = gNotDoneYetDescription,
+    [MOVE_MYSTICAL_POWER - 1] = gNotDoneYetDescription,
+    [MOVE_RAGING_FURY - 1] = gNotDoneYetDescription,
+    [MOVE_WAVE_CRASH - 1] = gNotDoneYetDescription,
+    [MOVE_CHLOROBLAST - 1] = gNotDoneYetDescription,
+    [MOVE_MOUNTAIN_GALE - 1] = gNotDoneYetDescription,
+    [MOVE_VICTORY_DANCE - 1] = gNotDoneYetDescription,
+    [MOVE_HEADLONG_RUSH - 1] = gNotDoneYetDescription,
+    [MOVE_BARB_BARRAGE - 1] = gNotDoneYetDescription,
+    [MOVE_ESPER_WING - 1] = gNotDoneYetDescription,
+    [MOVE_BITTER_MALICE - 1] = gNotDoneYetDescription,
+    [MOVE_SHELTER - 1] = gNotDoneYetDescription,
+    [MOVE_TRIPLE_ARROWS - 1] = gNotDoneYetDescription,
+    [MOVE_INFERNAL_PARADE - 1] = gNotDoneYetDescription,
+    [MOVE_CEASELESS_EDGE - 1] = gNotDoneYetDescription,
+    [MOVE_BLEAKWIND_STORM - 1] = gNotDoneYetDescription,
+    [MOVE_WILDBOLT_STORM - 1] = gNotDoneYetDescription,
+    [MOVE_SANDSEAR_STORM - 1] = gNotDoneYetDescription,
+    [MOVE_LUNAR_BLESSING - 1] = gNotDoneYetDescription,
+    [MOVE_TAKE_HEART - 1] = gNotDoneYetDescription,
```

Comment:
Can you please add move descriptions?

Reply:
Done. A lot of the descriptions I wrote probably suck big fat oranges though, so suggestions are welcomed.